scholia/app/templates/sparql/author/publications.sparql

```diff
@@ -0,0 +1,25 @@
+#defaultView:Table
+SELECT
+  (MIN(?dates) AS ?date)
+  ?work ?workLabel
+  (GROUP_CONCAT(DISTINCT ?type_label; separator=", ") AS ?type)
+  (SAMPLE(?pages_) AS ?pages)
+  ?venue ?venueLabel
+  (GROUP_CONCAT(DISTINCT ?author_label; separator=", ") AS ?authors)
+  (CONCAT("../authors/", GROUP_CONCAT(DISTINCT SUBSTR(STR(?author), 32); separator=",")) AS ?authorsUrl)
```

Comment:
Note that this line was the offending one that needed to be resolved. Now the updates that were originally in the HTML file are here :)